content/id/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -0,0 +1,416 @@
+
+```shell
+cd /proc/1
+cat status
+```
+
+Tampilan menunjukkan _bitmap_ kemampuan untuk proses tersebut:
+
+```shell
+...
+CapPrm:	00000000aa0435fb
+CapEff:	00000000aa0435fb
+...
+```
+
+Bandingkan kemampuan dari kedua kontainers tersebut:
+
+```
+00000000a80425fb
+00000000aa0435fb
+```
+
+Dalam _bitmap_ kemampuan pada kontainer pertama, bit-12 dan ke-25 tidak diatur. Sedangkan dalam kontainer kedua,
+bit ke-12 dan ke-25 diatur. Bit ke-12 adalah kemampuan `CAP_NET_ADMIN`, dan bit-25 adalah kemampuan `CAP_SYS_TIME`.
+Lihatlah [capability.h](https://github.com/torvalds/linux/blob/master/include/uapi/linux/capability.h)
+untuk definisi dari konstanta kemampuan yang lainnya.
+
+{{< note >}}
+Konstanta kemampuan Linux memiliki format `CAP_XXX`. Tetapi ketika kamu memasukkan daftar kemampuan dalam manifes kontainer kamu, kamu harus menghilangkan bagian `CAP_` dari konstanta. Misalnya, untuk menambahkan `CAP_SYS_TIME`, masukkan `SYS_TIME` dalam daftar kemampuan kontainer kamu.
+{{< /note >}}
+
+## Memberikan label SELinux pada sebuah kontainer
+
+Untuk memberikan label SELinux pada sebuah kontainer, masukkan bagian `seLinuxOptions` pada
+bab `securityContext` dari manifes Pod atau kontainer kamu.
+Bagian `seLinuxOptions` adalah sebuah objek [SELinuxOptions](/docs/reference/generated/kubernetes-api/{{<param"version">}}/#selinuxoptions-v1-core).
+Berikut ini adalah contoh yang menerapkan tingkat (_level_) dari SELinux:
+
+```yaml
+...
+securityContext:
+  seLinuxOptions:
+    level: "s0:c123,c456"
+```
+
+{{< note >}}
+Untuk menetapkan label SELinux, modul keamanan SELinux harus dimuat terlebih dahulu pada sistem operasi dari _host_.
+{{< /note >}}
+
+## Diskusi
+
+Konteks keamanan untuk sebuah Pod berlaku juga untuk kontainer yang berada dalam Pod tersebut dan juga untuk
+_volume_ dari Pod tersebut jika ada. Khususnya untuk `fsGroup` dan `seLinuxOptions`
+akan diterapkan pada _volume_ seperti berikut ini:
+
+* `fsGroup`: _Volume_ yang mendukung manajemen kepemilikan akan dimodifikasi agar dapat dimiliki
+dan ditulis oleh ID group (GID) yang disebutkan dalam `fsGroup`. Lihatlah
+[Dokumen Desain untuk Manajemen Kepemilikan](https://git.k8s.io/community/contributors/design-proposals/storage/volume-ownership-management.md)
+untuk lebih detail.
+
+* `seLinuxOptions`: _Volume_ yang mendukung pelabelan SELinux akan dilabel ulang agar dapat diakses
+oleh label yang ditentukan dari `seLinuxOptions`. Biasanya kamu hanya
+perlu mengatur bagian `level`. Dimana ini akan menetapkan label
+[Keamanan multi-kategori (_Multi-Category Security_) (MCS)](https://selinuxproject.org/page/NB_MLS)
+yang diberikan kepada semua kontainer dalam Pod serta Volume yang ada didalamnya.
+
+{{< warning >}}
+Setelah kamu menentukan label MCS untuk Pod, maka semua Pod dengan label yang sama dapat mengakses _volume_. Jika kamu membutuhkan perlindungan antar Pod, kamu harus menetapkan label MCS yang unik untuk setiap Pod.
+{{< /warning >}}
+
+## Membersihkan (_Clean Up_)
```

Comment:
```suggestion
## Bersih-bersih (_Clean Up_)
```

Let's use "bersih-bersih" instead. It sounds more friendly. 😄

Reply:
Agree!